go.mod

```diff
@@ -3,248 +3,243 @@ module github.com/dapr/cli
 go 1.20
 
 require (
-	github.com/Azure/go-autorest/autorest v0.11.28 // indirect
-	github.com/Azure/go-autorest/autorest/adal v0.9.22 // indirect
 	github.com/Pallinder/sillyname-go v0.0.0-20130730142914-97aeae9e6ba1
-	github.com/briandowns/spinner v1.19.0
-	github.com/dapr/dapr v1.11.0
+	github.com/briandowns/spinner v1.23.0
 	github.com/dapr/go-sdk v1.6.0
-	github.com/docker/docker v20.10.21+incompatible
+	github.com/docker/docker v23.0.3+incompatible
 	github.com/fatih/color v1.15.0
-	github.com/gocarina/gocsv v0.0.0-20220927221512-ad3251f9fa25
-	github.com/hashicorp/go-retryablehttp v0.7.1
+	github.com/gocarina/gocsv v0.0.0-20230616125104-99d496ca653d
+	github.com/hashicorp/go-retryablehttp v0.7.4
 	github.com/hashicorp/go-version v1.6.0
 	github.com/mitchellh/go-ps v1.0.0
 	github.com/nightlyone/lockfile v1.0.0
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/shirou/gopsutil v3.21.11+incompatible
-	github.com/spf13/cobra v1.6.1
-	github.com/spf13/viper v1.13.0
-	github.com/stretchr/testify v1.8.3
-	golang.org/x/sys v0.8.0
+	github.com/spf13/cobra v1.7.0
+	github.com/spf13/viper v1.16.0
+	github.com/stretchr/testify v1.8.4
+	golang.org/x/sys v0.12.0
 	gopkg.in/yaml.v2 v2.4.0
-	helm.sh/helm/v3 v3.11.1
-	k8s.io/api v0.26.3
-	k8s.io/apiextensions-apiserver v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/cli-runtime v0.26.3
-	k8s.io/client-go v0.26.3
```

Comment:
We still want to support 26.2 for this release can be updated in the next release . There is an issue tracking it.

Reply:
I have pushed a commit to update the runtime to rc.2 and have reverted the k8s package update in this commit - https://github.com/dapr/cli/pull/1342/commits/a139d396ff404ffd83032c4f6e4245604dfac3cb

let me know what you think ? @JoshVanL @mukundansundar

Reply:
@pravinpushkar please try not to push to authors branches, and instead favor code review or PRing to their branch :slightly_smiling_face:

Reply:
@JoshVanL Sure, thought you will not be able to look into it until my night time. So, was only trying to fast pace the process for RC cut.